tests/robustness/client/watch.go

```diff
@@ -48,7 +48,6 @@ func CollectClusterWatchEvents(ctx context.Context, param CollectClusterWatchEve
 			if err != nil {
 				return err
 			}
-			defer c.Close()
```

Comment:
I think the Close is called correctly. The ClientSet is implemented wrongly.

User stories:
* I want to ensure that client connections are cleaned up even if panic occurs. Aka `defer c.Close()` should work.
* I want reports generated by `c.Report()` to be complete. Aka connection should be closed before that.

Reply:
Hmm ok. 

Let me look into it. Thanks for the input.